src/modules/esl-toggleable/core/esl-toggleable.ts

```diff
@@ -314,8 +323,17 @@ export class ESLToggleable extends ESLBaseElement {
 
   /** Actions to execute on show request */
   @bind
-  protected _onShowRequest(): void {
-    this.show();
+  protected _onShowRequest(e: CustomEvent<ESLShowRequestDetails>): void {
+    const target = e.target;
+    const detail = e.detail;
+    if (!target) return;
+    let showOptions;
```

Comment:
Oh do we really need a mutable style here?
`let` usually indicates that the code will be overcomlicated instead of splited on the named immutable parts.